kubernetes/database.libsonnet

```diff
@@ -41,4 +41,28 @@ local k = import 'kubernetes/kube.libsonnet';
       instance_class: if std.objectHas(this.instance_classes, namespace) then this.instance_classes[namespace] else this.instance_classes['default'],
     },
   },
+  WaitForDatabaseProvisioning(database_cluster_name, namespace):  k._Object('databases.outreach.io/v1', 'PostgresqlDatabaseCluster', name=database_cluster_name, namespace=namespace) {
```

Comment:
One concern with the object versioning here, we probably will have different versions of the PostgresDatabaseCluster to wait for. This is not critical now, I'm good with leaving this as is just want to call it out.

Reply:
I believe the same issue exists with PostgresDatabaseCluster method as well. I guess we would upgrade both if we ever need to do an update.